### DISCUSSION TITLE
Documentation: using "proportional" to describe code (aka mono-spaced) fonts

### DISCUSSION THREAD

**Discussion by cubranic** [squidfunk/mkdocs-material#2309] [General]
I am confused by the use of "proportional" to describe the code font. (See [Changing the fonts](https://squidfunk.github.io/mkdocs-material/setup/changing-the-fonts/#proportional-font) page in the documentation.) That seems to mean the exact opposite of what a proportional font is in typography: a font in which characters ("glyphs") can be of varying widths. (See also [Wikipedia's description](https://en.wikipedia.org/wiki/Typeface#Proportion).)

I'd be happy to open an issue/PR with changed wording, but I am surprised that no-one has already brought it up, although I couldn't find any mention of it in closed issues. Am I missing something obvious here?

**Comment by squidfunk** [comment]
You're absolutely right, I think I swapped those concepts. A PR is very welcome, let's just call them monospaced fonts 😉

**Reply by cubranic** [reply]
Thanks @squidfunk! PR submitted in #2314.
